core/src/main/java/hudson/model/listeners/ItemListener.java

```diff
@@ -197,6 +197,14 @@ public static void checkBeforeCopy(final Item src, final ItemGroup parent) throw
     }
 
     public static void fireOnCreated(final Item item) {
+        // Do not use Listeners.notify here as impls in matrix-auth expect the Authentication to be retained.
+        for (ItemListener l : all()) {
+            try {
+                l.onCreated(item);
+            } catch (Throwable x) {
+                LOGGER.log(Level.WARNING, null, x);
+            }
+        }
         Listeners.notify(ItemListener.class, l -> l.onCreated(item));
```

Comment:
```suggestion
```

Reply:
I think this should be deleted now that we are restoring the old version of the code.